tests/e2e/scenarios/test_reborn_webui_v2_smoke.py

```
@@ -516,6 +516,137 @@ async def test_reborn_v2_automation_rename_persists_from_ui(
         assert renamed["automation_id"] == automation_id
 
 
+async def test_reborn_v2_automation_action_error_toast_is_safe_dismissible_and_cleared_on_retry(
```

Ruff (0.15.21)

[warning] Too many statements (61 > 50)

(PLR0915)